src/components/ui/PoweredByBanner.tsx

```diff
@@ -48,7 +48,7 @@ function Banner() {
   const { chatBottomContent } = useConstantState();
 
   return (
-    <InnerContainer chatBottomBackgroundColor={chatBottomContent.backgroundColor}>
+    <InnerContainer chatBottomBackgroundColor={chatBottomContent.backgroundColor} dir={'ltr'}>
```

Comment:
is constant `dir={'ltr'}` intended?

Reply:
Yeah, it's a `powered by sendbird` banner 
<img width="182" alt="image" src="https://github.com/user-attachments/assets/07b521d4-5968-4fe8-8c1c-a361cff2c68a">